Services (#243)
northeastern-rc/rc-public-documentation: docs/source/welcome/services/collaboration.md

(collaboration)=

# Collaboration

> The team at RC @ NU is composed of full-time PhD scientists and domain experts whose expertise spans diverse fields (e.g., Machine Learning, Data Analytics, Molecular Dynamics, Bioinformatics, and Big Data). We are eagerly open to collaboration with researchers in multiple phases of their research including:

Reviewer: ```suggestion
The team at RC @ NU is composed of full-time PhD scientists and domain experts whose expertise spans diverse fields (e.g., Machine Learning, Data Analytics, Molecular Dynamics, Bioinformatics, and Big Data). We are eagerly open to collaboration with researchers in multiple phases of their research, including:
```

> The team at RC @ NU is composed of full-time PhD scientists and domain experts whose expertise spans diverse fields (e.g., Machine Learning, Data Analytics, Molecular Dynamics, Bioinformatics, and Big Data). We are eagerly open to collaboration with researchers in multiple phases of their research including:

Author: Just made these changes! Thank you!

- Container building/development
- Course Development
- Data analysis
- Data cleaning and transforming
- Data visualization (exploratory and publication quality)
- Final publication preparation
- Grant/proposal development and idea generation
- HPC workflow optimization
- Reproducibility
- Statistical modeling

> Please reach out via a consultation or by emailing rchelp@northeastern.edu if you're interested in any of the above (or more) services.

Reviewer 2: ```suggestion
Please schedule a consultation or email rchelp@northeastern.edu if you're interested in any of the above (or more) services.
```
Can we add the bookings link to "schedule a consultation?"
